src/EditorServicesCommandSuite/CodeGeneration/Refactors/CommandSplatRefactor.cs

```diff
@@ -122,11 +325,19 @@ internal override async Task<IEnumerable<DocumentEdit>> RequestEdits(DocumentCon
             var splatVariable = string.IsNullOrWhiteSpace(config.VariableName)
                 ? GetSplatVariableName(ast.CommandElements.First())
                 : config.VariableName;
+            var executionContext =
+                CommandSuite
+                    .Instance
+                    .ExecutionContext;
```

Comment:
This can be inlined if it's only used as an argument to `GetEdits`.

Reply:
Ok